functional_tests.go

```diff
@@ -283,57 +367,70 @@ func testPutObjectReadAt() {
 
 	// Save the data
 	objectName := randString(60, rand.NewSource(time.Now().UnixNano()), "")
+	args["objectName"] = objectName
+
 	// Object content type
 	objectContentType := "binary/octet-stream"
 
```

Comment:
`args["objectContentType"]` needs to be populated with `objectContentType` value, since `objectContentType` is one of the parameters for `PutObject` function.